blog.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <meta charset="utf-8">
+      <title>Blog</title>
+      <link href="normalize.css" rel="stylesheet">
+      <link href="styles/blog.css" rel="stylesheet">
+    </head>
+      <body class="container">
+        <section class="header">
+          <li><a href="index(port).html">home.</a></li>
```

Comment:
Also `li` can't be a child of section, it can only be a child element of `ul` or `ol`.